experiment/measurer.py

```diff
@@ -92,6 +92,34 @@ def get_coverage_infomation(coverage_summary_file):
         return json.loads(summary.readlines()[-1])
 
 
+def store_profdata_files(experiment_config: dict):
```

Comment:
this function should be 
get_fuzzer_benchmark_trial_pairs
and then below function should be
store_coverage_profdata_files